.chloggen/fix_update-test-in-kafka-topic-observer.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
No need for a changelog for a test fix. Please delete this file.

Reply:
Done